test/intl402/PluralRules/prototype/resolvedOptions/return-keys-order-default.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.pluralrules.prototype.resolvedoptions
+description: order of property keys for the object returned by resolvedOptions()
+features: [Intl.NumberFormat-v3]
+includes: [compareArray.js]
+---*/
+
+let expected = [
+    "locale",
+    "type",
+    "minimumIntegerDigits",
+    "minimumFractionDigits",
+    "maximumFractionDigits",
+    "pluralCategories",
+    "roundingIncrement",
+    "roundingMode",
+    "roundingPriority",
+    "trailingZeroDisplay"
+];
```

Comment:
cf. https://github.com/tc39/test262/pull/3911#discussion_r1325120596
Please apply a technique like that in test/intl402/NumberFormat/prototype/resolvedOptions/return-keys-order-default.js to cover ordering of all `resolvedOptions()` properties (currently missing: "minimumSignificantDigits" and "maximumSignificantDigits").

Reply:
PTAL